core/templates/services/debouncer.service.spec.ts

```diff
@@ -20,9 +20,9 @@ import { DebouncerService } from 'services/debouncer.service';
 import { LoggerService } from 'services/contextual/logger.service';
 
 describe('Debouncer service', () => {
-  let ds;
-  let ls;
-  var loggerServiceSpy;
+  let ds: DebouncerService;
+  let ls: LoggerService;
+  let loggerServiceSpy: ReturnType<typeof spyOn>;
```

Comment:
Ditto*

Reply:
Same Explanation as above.

Reply:
Please check the above comment.

Reply:
@DubeySandeep I made some changes in the latest commit. PTAL!